code/modules/mob/living/carbon/human/_species.dm

```diff
@@ -1217,7 +1220,7 @@ GLOBAL_LIST_EMPTY(features_by_species)
 	// Core to skin temp transfer, when not on fire
 	if(!humi.on_fire)
 		// Get the changes to the skin from the core temp
-		var/core_skin_diff = humi.coretemperature - humi.bodytemperature
+		var/core_skin_diff = humi.coretemperature - original_bodytemp
```

Comment:
You are just reintroducing the body temp jank I removed specifically due to buggy implementation while adding temperature regulation to lizards. This is a species change without maintainer approval which is required for things like these. Its not a fix, its balance too.

Reply:
I explained in my last PR that the original code is wrong and causes the body temp to get stuck. What 'jank' are you referring to with my fix?

Reply:
Fixed the changelog

Reply:
> species change without maintainer approval

Species changes need headcoder approval, not maintainer.